src/ucp/core/ucp_worker.c

```diff
@@ -2413,6 +2415,24 @@ ucs_status_t ucp_worker_query(ucp_worker_h worker,
         attr->max_am_header = ucp_am_max_header_size(worker);
     }
 
+    if (attr->field_mask & UCP_WORKER_ATTR_FIELD_BANDWIDTH) {
+        attr->bandwidth = 0;
+
+        UCS_BITMAP_FOR_EACH_BIT(context->tl_bitmap, tl_id) {
```

Comment:
Why is bandwidth related to address flags? If we are requesting just NET devices it does not mean we will not use other devices (for instance if CS connection establishment used).
Worth to be described in API also